### PR TITLE
Server improvements

### PR DESCRIPTION
**Pull request by koriakin**

### CODE REVIEW

File: python/veles/scli/client.py [hunk 1]
```diff
@@ -38,6 +39,20 @@ def getpkt(self):
...
+    def request(self, msg):
+        self.send_msg(msg)
+        pkt = self.getpkt()
+        if isinstance(pkt, messages.MsgRequestAck) and pkt.rid == 0:
+            return msg.id
+        elif isinstance(pkt, messages.MsgRequestError) and pkt.rid == 0:
+            raise VelesException(pkt.code, pkt.msg)
+        else:
+            print(pkt)
+            raise Exception('weird reply to create')
```
Comment (koriakin) [inline python/veles/scli/client.py]: Fixed.

File: python/veles/proto/messages.py [hunk 2]
```diff
@@ -177,30 +236,88 @@ class MsgRegistryReply(MsgpackMsg):
...
+    object_type = 'query_error'
+
+    qid = fields.SmallUnsignedInteger()
+    code = fields.String()
+    msg = fields.String()
+
+
+class MsgCancelSubscription(MsgpackMsg):
+    """
+    Sent by the cleint.  Cancels a subscription with the given qid.
```
Comment (altran01) [inline python/veles/proto/messages.py]: nit. typo :)

File: python/veles/proto/messages.py [hunk 3]
```diff
@@ -226,6 +353,11 @@ class MsgAddTag(MsgpackMsg):
 
 
 class MsgDelTag(MsgpackMsg):
+    """
+    Removes a tag from a given node.  Server replies with MsgRequestAck
```
Comment (altran01) [inline python/veles/proto/messages.py]: Maybe add note that deleting non-existing tag won't result in error?

File: python/veles/proto/messages.py [hunk 4]
```diff
@@ -70,91 +87,133 @@ class MsgRegisterTrigger(MsgpackMsg):
...
 
-    id = fields.NodeID()
     qid = fields.SmallUnsignedInteger()
+    id = fields.NodeID()
     sub = fields.Boolean(default=False)
 
 
 class MsgGetReply(MsgpackMsg):
+    """
+    Sent by server in reply to MsgGet.
```
Comment (koriakin) [inline python/veles/proto/messages.py]: I've thrown in documentation for the Node structure.

File: python/veles/proto/messages.py (hunk 3 above)
Comment (koriakin) [inline python/veles/proto/messages.py]: Done.

File: python/veles/proto/messages.py (hunk 2 above)
Comment (koriakin) [inline python/veles/proto/messages.py]: Fixed.

**Pull request merged**